### DISCUSSION TITLE
Install error in CentOS 7

### DISCUSSION THREAD

**Discussion by chuang05** [idaholab/moose#28910] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

I'm install moose using conda in CentOS 7, this error encountered, how to solve this problem?
```
(base) [user@server96 ~]$ conda create -n moose moose-dev=2024.10.01=mpich
Channels:
 - conda-forge
 - https://conda.software.inl.gov/public
 - defaults
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - nothing provides __glibc >=2.28,<3.0.a0 needed by c-ares-1.33.1-heb4867d_0

Could not solve for environment specs
The following package could not be installed
└─ moose-dev ==2024.10.01 mpich is not installable because it requires
   └─ moose-libmesh 2024.08.30 mpich_1, which requires
      └─ moose-petsc 3.21.5 mpich_0, which requires
         └─ moose-mpi 2024.10.01.* , which requires
            └─ moose-mpi-base [2024.10.01 mpich|2024.10.01 openmpi], which requires
               └─ c-ares 1.33.1 heb4867d_0, which requires
                  └─ __glibc >=2.28,<3.0.a0 , which is missing on the system.

```

**Comment by GiudGiud** [comment]
Hello

Can you try relaxing the constraint on the version and see which version you end up with ?

That constraint is at the end of the conda install command after the package name

Guillaume

**Comment by milljm** [comment]
The version of Conda you have installed may be too new for the operating system you are running. There isn't going to be much we can do here...

MOOSE it self will still run, but you may be required to build your own compiler stack. Or use the one provided by the system. Honestly that is not as hard as it sounds.

You'll want to start searching online for "centos install developer tools". And/Or, start by following our manual install instructions: https://mooseframework.inl.gov/getting_started/installation/manual_installation_gcc.html

**Reply by chuang05** [reply]
Thank you for your help, I will try again.
